LibVLCSharp.Uno/VideoView.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using LibVLCSharp.Shared;
+using Windows.ApplicationModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace LibVLCSharp.Platforms.UWP
+{
+    /// <summary>
+    /// Video view
+    /// </summary>
+    public partial class VideoView : Control, IVideoView
+    {
+        /// <summary>
+        /// Occurs when the <see cref="VideoView"/> is fully loaded
+        /// </summary>
+        public event EventHandler<InitializedEventArgs>? Initialized;
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="VideoView"/> class
+        /// </summary>
+        public VideoView()
+        {
+            DefaultStyleKey = typeof(VideoView);
+            Loaded += (sender, e) => Initialized?.Invoke(this, new InitializedEventArgs(SwapChainOptions));
+        }
+
+        private Border? Border { get; set; }
+
+        /// <summary>
+        /// Gets the swapchain parameters to pass to the <see cref="LibVLC"/> constructor.
+        /// Calling this property will throw an <see cref="InvalidOperationException"/> if the VideoView is not yet fully loaded.
+        /// </summary>
+        /// <returns>The list of arguments to be given to the <see cref="LibVLC"/> constructor</returns>
+        public string[] SwapChainOptions
+        {
+            get
+            {
+                if (!IsLoaded)
+                {
+                    throw new InvalidOperationException("You must wait for the VideoView to be loaded before calling GetSwapChainOptions()");
+                }
+
+                return Array.Empty<string>();
```

Comment:
This property either throws or returns an empty array. Can we do without?

Reply:
No, I can remove the `throw` but I must let this property. The goal is to be able to use exactly the same code with UWP, Android or iOS.